policies/release-requirements.md

```diff
@@ -0,0 +1,92 @@
+Release Requirements Policy
+===========================
+
+The OpenSSL project team creates the following three types of OpenSSL software
+releases:
+
+- [stable] releases
+- [alpha] (pre-)releases
+- [beta] (pre-)releases
+
+This policy defines the requirements on the state of a branch in the source
+tree that must be met before a release from that branch can be done.
+
+Alpha Pre-releases
+------------------
+
+As this is just a preview release for testing things that have been worked
+on in the development branch, the requirements are minimal.
+
+- The CI must pass on the tip of the development branch before the release
+  commits were added to the tree.
+
+Beta Pre-releases
+-----------------
+
+The API and ABI should be stable and the source code should be feature complete
+by the first beta pre-release. The following release requirements apply to beta
+pre-releases.
+
+- The code is functionaly complete in regards to the particular release
+  objectives as set by OMC and OTC.
+- There is no remaining refactoring required by OMC or OTC for the release.
+- There are no remaining API changes required for the release.
+  _This applies only to beta releases of a major release as API changes
+  are not allowed on minor releases._
+- There are no remaining API additions required for the release.
+- All untriaged issues older than 2 days must be triaged.
+- All regressions on the development branch from which the
+  release is to be done must have a milestone assigned by OTC.
+  _Regressions are considered from the previous stable release. The process
+  how OTC assigns the milestone is not exactly defined on purpose._
+- All issues or pull requests with beta milestone for the beta release
+  are closed.
```

Comment:
It's not really clear what this means to me. Does that mean all bugs must be fixed before the next beta release?

Reply:
This supposes that there exists a beta milestone for each beta release. Then when bugs are triaged the individual beta milestones are set. So no, it does not mean all bugs must be fixed, just those with the beta milestone for this beta release that's being released.